<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-5JP91O4B" failures="1" tests="7" name="Tests" time="8.224" errors="0" timestamp="2021-09-04T20:21:18 CEST">
  <testcase classname="com.org.tests.APITest" name="getAllRepo" time="1.141">
    <failure type="java.lang.AssertionError" message="Total count changed as this is a dynamic value expected [643292] but found [643294]">
      <![CDATA[java.lang.AssertionError: Total count changed as this is a dynamic value expected [643292] but found [643294]
at com.org.tests.APITest.getAllRepo(APITest.java:47)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- getAllRepo -->
  <testcase classname="com.org.tests.APITest" name="getRepoByLanguage" time="1.271"/>
  <testcase classname="com.org.tests.APITest" name="getRepoByDate" time="0.552"/>
  <testcase classname="com.org.tests.APITest" name="getRepoByTwoLanguages" time="1.238"/>
  <testcase classname="com.org.tests.APITest" name="getRepoByUser" time="0.741"/>
  <testcase classname="com.org.tests.APITest" name="getStarredRepo" time="1.244"/>
  <testcase classname="com.org.tests.APITest" name="dataNotFound" time="0.203"/>
</testsuite> <!-- Tests -->
